rfcs/0173-hotline.md

```diff
@@ -0,0 +1,234 @@
+
+```
+I understood MicNinetyThree, is this correct?
+```
+
+- User says no
+```
+Could you please repeat the name?
+```
+
+- User says Mic92
+
+```
+I understood MicNineThree, is this correct?
+```
+
+- User says no
+
+```
+Sadly all our speech to text pipelines are busy. Instead we will read to you all pull requests by r-ryantm.
+
+- pull request number 299112: Python 3 point eleven AIO ESP Home API. twentythree point one point zero arrow twentythree point two point zero
+- pull request number 299106: D DNS dash go. six point two point two arrow six point three point zero
+(etc.)
+
+- please press the number of the pull request you  want to edit.
+```
+
+- User presses 299106
+
+```
+welcome to the pull request 299106
+please press 1 to listen to all comments.
+press 2 to close this pull request.
+press 3 to merge this pull request.
+press 4 to leave a review.
+```
+
+- User presses 3
+
+```
+Please enter your private key on the dialpad. Take care, that the private key has to be converted to base 10 first.
+```
+
+- User presses 111 112 101 110 115 115 104 45 107 101 121 45 118 49 0 0 0 0 4 110 111 110 101 0 0 0 4 110 111 110 101 0 0 0 0 0 0 0 1 0 0 0 51 0 0 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 1014 80 41974 80 40960 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 64 2399 1243 115 101 112 89 27 34727 33 107 6827 64594 80 86 80 125 43 10 36 49 9295 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 12 108 97 115 115 64 105 103 110 97 118 105 97 2
+
+```
+sadly the private key you entered is incorrect, please try again.
+```
+
+- User presses 111 112 101 110 115 115 104 45 107 101 121 45 118 49 0 0 0 0 4 110 111 110 101 0 0 0 4 110 111 110 101 0 0 0 0 0 0 0 1 0 0 0 51 0 0 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 1014 80 41974 80 40960 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 64 2399 1243 115 101 112 89 27 34727 33 107 6827 64594 80 86 80 125 43 10 36 49 9295 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 12 108 97 115 115 64 105 103 110 97 118 105 97 1
+
+```
+Thank you, the pull request 299106 was merged into the master branch. see you later aligator.
+```
+
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+None are obvious, this seems like a perfect system.
+
+# Alternatives
+[alternatives]: #alternatives
+
+Fax, Letters, pidgeons. These are not really alternatives though as they are more of addons.
+
+Human operator fallback.
+
+The ability to go to a physical office, but only on the fourth Friday in uneven months between 7:30 and 8:15 am with an appointment via hotline.
```

Comment:
To ensure our support teams can properly schedule the numerous appointments during the day I believe users should make sure to plan theirs in advance
```suggestion
The ability to go to a physical office, but only on the fourth Friday in uneven months between 7:30 and 8:15 am with an appointment via hotline requested at least 2 months in advance.
```